x-pack/plugins/cloud_security_posture/public/pages/rules/rules.test.tsx

```diff
@@ -78,13 +82,14 @@ describe('<Rules />', () => {
     const Component = getTestComponent({ packagePolicyId: '21', policyId: '22' });
     const request = {
       status: 'loading',
+      isLoading: true,
     };
 
     (useCspIntegration as jest.Mock).mockReturnValue(request);
 
     render(<Component />);
 
-    expect(screen.getByTestId(cspLoadingStateTestId)).toBeInTheDocument();
+    expect(screen.queryByTestId(cspLoadingStateTestId)).toBeInTheDocument();
```

Comment:
why did we change this?  i think it should be `getByTestId` 

seems like docs say `getByTestId` would throw if not found, and this would return `null`

there's a (very weird) case where this test passes if it finds `null` in the document.